flake.nix

```diff
@@ -37,7 +37,7 @@
                     ps
 
                     # typescript
-                    python3
+                    python311
```

Comment:
This is required to build `lodestar`. Python version 3.12 deprecates some required packages.